subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ if(CONFIG_BT_LL_SW_SPLIT)
     zephyr_library_sources_ifdef(
         CONFIG_BT_CTLR_PERIPHERAL_ISO
         ll_sw/ull_peripheral_iso.c
+        ll_sw/ull_iso.c
```

Comment:
This should go somewhere else. On the one hand, I think it should be added to the sources only one. On the other hand, also BIS roles would need this file.

Reply:
Fixed